src/Components/getByModelForMake.js

```diff
@@ -0,0 +1,84 @@
+import React, { useEffect, useState } from 'react';
+import { connect, useSelector } from 'react-redux';
+import { Link } from 'react-router-dom';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faCarSide } from '@fortawesome/free-solid-svg-icons';
+import carImg from '../images/black-car3.png';
+import { VehicleByModelForMake } from '../redux/actions';
+
+function GetByModelForMake({ getVehiclesData }) {
+  useEffect(() => {
+    getVehiclesData(440);
+  }, []);
+  const [page, setPage] = useState(0);
+  const [search, setSearch] = useState('');
+  let numPages = 0;
+  const vehicles = useSelector((state) => state.vehiclesReducer);
+  const numOfpages = Math.floor(vehicles.length / 20);
+  if (vehicles.length % 20 === 0) {
+    numPages = numOfpages;
+  } else {
+    numPages = numOfpages + 1;
+  }
+
+  const dropvalue = (e) => {
+    setSearch(e.target.value);
+  };
+
+  const submitUrl = (e) => {
+    if (e.keyCode === 13 && search.length > 0) {
+      getVehiclesData(search);
+    }
+  };
+
+  return (
+    <div className="child-page">
+      <Link to="../">Home</Link>
+      <img src={carImg} alt="" className="big-car-img" />
+      <div className="line" />
+      <h2>Vehicles Models for Makes</h2>
+      <input type="search" onChange={dropvalue} onKeyDown={submitUrl} placeholder="search Model by Id" />
```

Comment:
- [ ] [OPTIONAL] You have a lot of components like this one with repeated code. I understand that you are trying to fetch different results based on different searches and URL. Have you considered having a simple `Results.js` component which fetches and renders the appropriate results based on the URL?

If you're wondering how to go about that, `useLocation()` from `react-router-dom` would be really helpful. Please look it up and take advantage of it to reduce repetition in your code and make your project more maintainable. 😎